.github/workflows/build-lint-test.yml

```diff
@@ -10,6 +10,9 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
```

Comment:
This package never had a proper Node.js setup step. The default Node.js version now seems to be `18`, causing installation of dependencies to fail.